### PR TITLE
Uipackages

### PR DESCRIPTION
**Pull request by realswat1**
This is for all the dependencies of the packages needed to be imported later on for the react app and express app .

### CODE REVIEW

**Comment by alfonsocheng** [comment]
since you added index.html, you should also add a screenshot of it

File: exposee-app/Routes/user.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+// Route for user login
+router.post('/users/login', async (req, res) => {
+  const { username, password } = req.body;
+
+  try {
+    // Find the user by username
+    const user = await User.findOne({ where: { username } });
+
+    if (!user) {
+      return res.status(401).json({ error: 'Invalid username or password' });
```
Comment (alfonsocheng) [inline exposee-app/Routes/user.js:50]: this doesn't seem to make sense (why would it be invalid password?)

File: exposee-ui/src/App.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#root {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: center;
+}
+
+.logo {
```
Comment (alfonsocheng) [inline exposee-ui/src/App.css:8]: do you need all this logo boilerplate?

File: exposee-ui/src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import './App.css'
+
+function App() {
+  const [count, setCount] = useState(0)
+
+  return (
```
Comment (alfonsocheng) [inline exposee-ui/src/App.jsx:9]: do you need all this boilerplate?

File: exposee-ui/src/App.css (hunk 2 above)
Comment (realswat1) [reply exposee-ui/src/App.css:8]: i would be changing them. I got them from creating the vite package

**Pull request closed without merging**